templates/CRM/Civicase/Page/ContactCaseTab.tpl

```diff
@@ -8,15 +8,7 @@
       angular.module('civicaseContactTab').config(function($routeProvider) {
         $routeProvider.when('/', {
           reloadOnSearch: false,
-          resolve: {
-            hiddenFilters: function() {
-              return {
-                "contact_id": [{/literal}{$cid|json}{literal}]
-              };
-            }
-          },
-          controller: 'CivicaseCaseList',
-          templateUrl: '~/civicase/CaseList.html'
+          template: '<civicase-contact-case></civicase-contact-case>'
```

Comment:
We any way need to fetch the user ID as it will be dynamic and will be fetched from the URL.
Also I don't think we should pass filters as params as `ContactCaseTab` is the directive only used on contact case tab and there is no need to make this part of the directive configurable.

Reply:
We any way need to fetch the user ID as it will be dynamic and will be fetched from the URL.
Also I don't think we should pass filters as params as `ContactCaseTab` is the directive only used on contact case tab and there is no need to make this part of the directive configurable.